modules/mobile/spec/models/adapters/appointments_vaos_v2_adapter_spec.rb

```diff
@@ -913,5 +913,22 @@ def appointment_data(index = nil)
         expect(result.reason).to be_nil
       end
     end
+
+    context 'when some acheron field keys are camel case' do
+      it 'parses both camel case and non camel case fields' do
+        appointment = appointment_data[12]
+        appointment[:reason_code][:coding] = [{ code: 'will not be used' }]
```

Comment:
It took me a few minutes to understand that these values are essentially holdovers from the above test to confirm that the embedded values are used preferentially over these values.